### PR TITLE
fix(bw): multiple issues with menu navigation for 9X type radios

### PR DESCRIPTION
**Pull request by philmoz**
From 2.10 the menu navigation on 9X style radios (T-Lite, T12 and LR3Pro) does not work correctly.

Issues:
- Horizontal navigation on the SF/GF list does not work. Cannot select all properties for each SF/GF type.
- The FM list view has too many lines. Scrolling up from the top shows two blank lines at the bottom.
- Returning from the FM edit page to the FM list view may show the list view with the wrong scroll position if the edit page was scrolled to the bottom before exiting. Also affects other B&W radios when using long press to exit FM edit page.
- The short press of the EXIT key following a long press to exit a subpage was not being killed resulting in spurious activation of the EXIT key. Also affects other B&W radios.

Fixing the above issues caused other issues requiring more fixes (Model select page was one row short and FM edit page was missing G9 entry).

### CODE REVIEW

**Review by pfeerick — APPROVED** [review]
Seems good on LR3Pro and no apparent regressions on X9D+

File: radio/src/keys.h [hunk 1]
```diff
@@ -148,14 +148,14 @@ inline bool IS_KEY_EVT(event_t evt, uint8_t key)
...
-  return evt == EVT_KEY_FIRST(KEY_DOWN) || evt == EVT_KEY_REPT(KEY_DOWN) ||
+  return evt == EVT_KEY_FIRST(KEY_RIGHT) || evt == EVT_KEY_REPT(KEY_RIGHT) ||
          evt == EVT_KEY_FIRST(KEY_MINUS) || evt == EVT_KEY_REPT(KEY_MINUS) ||
          evt == EVT_ROTARY_RIGHT;
 }
 
 inline bool IS_PREVIOUS_EVENT(event_t evt)
 {
-  return evt == EVT_KEY_FIRST(KEY_UP) || evt == EVT_KEY_REPT(KEY_UP) ||
+  return evt == EVT_KEY_FIRST(KEY_LEFT) || evt == EVT_KEY_REPT(KEY_LEFT) ||
```
Comment (pfeerick) [inline radio/src/keys.h:158]: Actually, why did you change this? As up/down was correct for menu navigation on the t-lite (and also the LR3pro I think). Hence #4917

Comment (philmoz) [reply radio/src/keys.h:158]: To fix the left right scrolling on the SF/GF page for 9x nav - I fixed this first and meant to come back and check that it did not affect other things (which it clearly did). Thanks for picking it up - the fix in 4917 looks better.

Comment (pfeerick) [reply radio/src/keys.h:158]: Yeah, I thought that might be the case when digging a bit deeper...  I suspect it is has been broken somewhere else, but that should hopefully do for now to make most all the use cases work. :crossed_fingers:
